lib/Serialization/Serialization.cpp

```diff
@@ -1830,7 +1831,6 @@ static bool shouldSerializeMember(Decl *D) {
     if (D->getASTContext().LangOpts.AllowModuleWithCompilerErrors)
       return false;
     llvm_unreachable("decl should never be a member");
-
   case DeclKind::Missing:
```

Comment:
Nitpick: This chunk and the next one doesn't fit with the overall spacing style of this code. If you get back to this code or have to update this PR it would be nice to make it uniform.